.github/workflows/Dockerfile.yml

```diff
@@ -16,7 +16,7 @@ concurrency:
 
 env:
   HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
-  IMAGE: ${{ format('{0}/{1}:{2}', secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io', github.repository, github.event_name == 'pull_request' && format('pr-{0}', github.event.pull_request.number) || 'latest') }}
+  IMAGE: ${{ format('{0}/stellar-system-test:{1}', secrets.DOCKERHUB_TOKEN && 'docker.io' || 'ghcr.io', github.event_name == 'pull_request' && format('pr-{0}', github.event.pull_request.number) || 'latest') }}
```

Comment:
This will result in trying to push `docker.io/stellar-system-test:...` which I don't think is what you want.

Reply:
@leighmcculloch , if you don't mind taking a brief second look, I have resolved in a new small pr, just a couple lines:
https://github.com/stellar/system-test/pull/7

that one validated docker push worked, can see the pr image tagged on [dockerhub/stellar-system-test](https://hub.docker.com/r/stellar/stellar-system-test/tags)